src/shared/universal.py

```diff
@@ -17,3 +19,7 @@ def format_plan_nickname(product_name: str, plan_interval: str) -> str:
         formatted_interval = plan_interval
 
     return f"{product_name} ({formatted_interval})".title()
+
+
+def get_indempotency_key() -> str:
```

Comment:
We will need to re-run performance around some of these cases.  This calls has performance implications.

Reply:
So are you going to re-run perfomance? Or are you going to teach me how?  I'm good either way.

Reply:
After this lands, we can walk thru how to do it.